cabal.project

```diff
@@ -219,8 +219,8 @@ source-repository-package
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/cardano-ledger
-  tag: e290bf8d0ea272a51e9acd10adc96b4e12e00d37
-  --sha256: 1pmdg80a8irrqgdhbp46a9jx628mjbrj0k89xv5nb5dy37z5ig5f
+  tag: 52da70e5a0472cd4433876289f1aebaa0c6e5c85
```

Comment:
Looks good: 52da70e5a0472cd4433876289f1aebaa0c6e5c85 is six commits ahead of e290bf8d0ea272a51e9acd10adc96b4e12e00d37